package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "prettier": "2.8.0"
```

Comment:
Why is prettier outside of the frontend and backend folders?

Reply:
Good question it's outside so that it can be run on both folders for style formatting